### PR TITLE
Update CallFlow_Dashboard.json

### PR DESCRIPTION
**Pull request by tony1661**
Fixed the Query for the Call Flow panel. There was a newline that was causing the query in Grafana to error out.

I've added some screenshots to best display the issue:

---

### **Error on Dashboard:**
image

---

### **Fixed:**
image

### CODE REVIEW

File: all-in-one/grafana/provisioning/dashboards/CallFlow_Dashboard.json [hunk 1]
```diff
@@ -190,7 +190,7 @@
             "uid": "loki"
           },
           "editorMode": "code",
-          "expr": "{job=\"heplify-server\", src_ip=~\"$src_ip\", dst_ip=~\"$dst_ip\"} | response=~\"$method\" or type!=\"sip\" |~ `${search:raw}` | regexp \"Call-ID:\\s+(?<callid>.+?\r\n)\"",
+          "expr": "{job=\"heplify-server\", src_ip=~\"$src_ip\", dst_ip=~\"$dst_ip\"} | response=~\"$method\" or type!=\"sip\" |~ `${search:raw}` | regexp \"Call-ID:\\s+(?<callid>.+?)\"",
```
Comment (lmangani) [inline all-in-one/grafana/provisioning/dashboards/CallFlow_Dashboard.json]: This is a step in the right direction but we need to make sure the match stops at a newline for this to work

Comment (fergusean) [reply all-in-one/grafana/provisioning/dashboards/CallFlow_Dashboard.json]: in that case, @tony1661 I believe this is what you're looking for and fixes the issue for me:
```suggestion
          "expr": "{job=\"heplify-server\", src_ip=~\"$src_ip\", dst_ip=~\"$dst_ip\"} | response=~\"$method\" or type!=\"sip\" |~ `${search:raw}` | regexp \"Call-ID:\\s+(?<callid>.+?\\r\\n)\"",
```

**Comment by tony1661** [comment]
@lmangani I believe we have the solution thanks to the help of @fergusean

**Comment by lmangani** [comment]
Thank you so much @tony1661 @fergusean for your kind contribution! We appreciate it lots!

**Pull request merged**